tests/Integration/AlgoliaIntegrationTestCase.php

```diff
@@ -0,0 +1,135 @@
+<?php
```

Comment:
You should rebuild a test suite, and make it similar to the CTS ( kind of ). I am going to discuss this with you personally.